app/services/default_values_object.rb

```diff
@@ -0,0 +1,21 @@
+class DefaultValuesObject
```

Comment:
Okay, these are just my opinions, but here goes.

This is a service object which takes a single parameter, which there's no API to access nor would you want to, for instantiation. There are no methods that reference it, so you're not simplifying a method signature or allowing for injected dependencies. Thus, this class is effectively equivalent to

```ruby
class DefaultValuesObject
  def set_default_values(input_object)
    default_values.each_pair do |key, value|
      input_object[key] = value if input_object[key].first.empty?
    end
    input_object
  end
end
```

Now the API is default_values_object.set_default_values(@edit_form)

Since you've rewritten it this way, you can see the classic anti-pattern of feature envy (https://sourcemaking.com/refactoring/feature-envy). This class basically becomes a hash holder for things that are always called on `input_object`

The other problem is this object has one method and no interface except that one method, which takes the arguments it needs for that method - that's a function, which is effectively namespaced by a constant (DefaultValuesObject). It would be just as effective if this was `DefaultValuesObject.set_default_values(input_object)`, which is just a function - and ruby's OO.